docs/src/development/ssh.md

```diff
@@ -2,131 +2,118 @@
+A key pair is valid for as long as you have access to the private key on the system from which you are connecting. If you have a key pair available, you are not prompted to login.
 
-A keypair is valid for as long as you have access to the private key on the system from which you are connecting.  If you have a keypair available you will not be prompted to login.
+If you used GitHub to sign up for your Platform.sh account, your public keys from GitHub are automatically synced to your Platform.sh account. So you can use them already with the CLI or to [connect to a server](#connect-to-your-server-with-SSH).
 
-### Find your Public-Private keypair
+Otherwise, you might be able to [find existing keys](#find-your-keys) or else you need to [generate new keys](#generate-new-keys).
 
-If you use Linux, you probably already have keys. The private key is usually in a file named `~/.ssh/id_rsa` and the public key in `~/.ssh/id_rsa.pub`.
+### Find your keys
 
-Searching for a public key file:
-1. Open up a command prompt.
-2. Run the following commands:
+If you use Linux, you probably already have keys. The private key is usually in a file named `~/.ssh/id_rsa` and the public key in `~/.ssh/id_rsa.pub`. Tech Republic has a guide to [finding keys on different systems](https://www.techrepublic.com/article/how-to-view-your-ssh-keys-in-linux-macos-and-windows/).
 
-    ```bash
-    $ cd ~/.ssh
-    $ ls -a
-    id_rsa
-    id_rsa.pub
-    known_hosts
-    authorized_keys
-    ```
+To find your public key file:
 
-    If you find a file named `id_rsa.pub`, you can use it with Platform.sh. If you don't find an existing key, see the steps to create a new one in the [next section](#create-a-new-public-private-keypair).
+1. Open a terminal.
+2. Run the following commands:
 
-### Create a New Public-Private Keypair
+   ```bash
+   $ cd ~/.ssh
+   $ ls -a
+   ```
 
-{{< note >}}
-If you already have a SSH keypair, you can skip this step.
-{{< /note >}}
+If you find a file ending in `.pub`, copy the location and [add it to your Platform.sh account](#add-an-ssh-key-to-your-platform-account).
 
-Create a public-private keypair:
+If you don't find an existing key, [generate new keys](#generate-new-keys).
 
-```bash
-$ ssh-keygen -t rsa -C "your_email_address@example.com"
-```
+### Add an SSH key to your Platform account
 
-`ssh-keygen` generates the key pair and will ask you where you want to save the file:
-
-```bash
-Generating public/private rsa key pair.
-Enter file in which to save the key (/Users/your_username/.ssh/id_rsa):
-```
+Once you have the location of your public key, add it to your Platform.sh account. This method uses the Platform.sh CLI similar to [authenticating with certificates](#authenticate-with-certificates). You can also add it in the management console, similar to this [video](https://docs.platform.sh/videos/management-console/add-ssh-mc.mp4).
```

Comment:
```suggestion
Once you have copied your public key, add it to your Platform.sh account. This method uses the Platform.sh CLI similar to [authenticating with certificates](#authenticate-with-certificates). You can also add it in the management console, similar to this [video](https://docs.platform.sh/videos/management-console/add-ssh-mc.mp4).
```

I'll most likely reverse the order of explanations here, if we look at these as 2 paths one would be: 

SSH Keys + Git + Console

Platform CLI + Git + Console

Reply:
I'm not sure I understood this, but please check if you think it's still an issue.